### PR TITLE
Make SVPH package optional

### PR DESCRIPTION
**Pull request by adayton1**
# Summary

- This PR is a feature
- It does the following:
  - Makes the SVPH package optional to reduce build times and library size
  - Filter out SVPH tests when SVPH is disabled

### ToDo :

- [x] Annotate ``RELEASE_NOTES.md`` with notable changes.
- [x] Create LLNLSpheral PR pointing at this branch. (PR# https://rzlc.llnl.gov/gitlab/owen/LLNLSpheral/-/merge_requests/95)
- [x] LLNLSpheral PR has passed all tests.

### CODE REVIEW

**Comment by adayton1** [comment]
For one application, this decreased the build time of spheral by 8 minutes and the library size by 200 MB.

File: src/SVPH/CMakeLists.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@ set(SVPH_inst
     SVPHFacetedHydroBase
     SVPHCorrectionsPolicy
     SVPHMassDensityPolicy
+    TensorSVPHViscosity
```
Comment (adayton1) [inline src/SVPH/CMakeLists.txt:11]: Should we consider renaming this file? Maybe SVPHViscosityTensor?

**Comment by adayton1** [comment]
I think I hit some tests that fail sporadically. They might just need to be rerun, but I don't have the permissions to relaunch CI jobs.

**Comment by jmikeowen** [comment]
I've restarted the Spheral tests, which I think are likely to pass.  The LLNLSpheral tests however are failing right off the bat -- it looks like you're not being allowed to run them?  Probably need to contact @mdavis36 and see what's going on there.

**Comment by adayton1** [comment]
@mdavis36 looks like we're hitting disk quota on some of these jobs now.

**Comment by mdavis36** [comment]
@adayton1 should be fixed

**Comment by adayton1** [comment]
@mdavis36, @ptsuji, @jmikeowen, this is ready for review.

**Review by ptsuji — APPROVED** [review]

File: src/SVPH/CMakeLists.txt (hunk 1 above)
Comment (adayton1) [reply src/SVPH/CMakeLists.txt:11]: Moving this back to ArtificialViscosity and leaving the name as is.

**Comment by adayton1** [comment]
I've reverted all changes to TensorSVPHViscosity since it does not depend on anything in SVPH.

**Comment by adayton1** [comment]
This is ready for another review pass.

**Review by jmikeowen — APPROVED** [review]
Looks good -- thanks for taking care of my suggested changes so quickly.

**Comment by adayton1** [comment]
> Looks good -- thanks for taking care of my suggested changes so quickly.

Of course! Thanks for your suggestions!

**Review by robinson96 — APPROVED** [review]
LGTM. Looking forward to this!

**Pull request merged**